python/mlc_chat/quantization/quantization.py

```diff
@@ -162,40 +163,42 @@ def quantize_weight(self, weight: tvm.runtime.NDArray) -> List[tvm.runtime.NDArr
         model_dtype="float16",
         linear_weight_layout="NK",
     ),
-    "fp8_e4m3_e5m2_max_calibration": GroupQuantize(
+    "fp8_e4m3_e5m2_max_calibration": PerTensorQuantize(
         name="fp8_e4m3_e5m2_max_calibration",
-        kind="group-quant",
-        group_size=64,
-        quantize_dtype="e5m2_float8",
+        kind="per-tensor-quant",
+        activation_dtype="e4m3_float8",
+        weight_dtype="e5m2_float8",
         storage_dtype="uint32",
         model_dtype="float16",
-        linear_weight_layout="NK",
+        no_scale=True,
+        quantize_embedding=False,
     ),
-    "fp8_e4m3_e5m2_max": GroupQuantize(
+    "fp8_e4m3_e5m2_max": PerTensorQuantize(
         name="fp8_e4m3_e5m2_max",
-        kind="group-quant",
-        group_size=64,
-        quantize_dtype="e5m2_float8",
+        kind="per-tensor-quant",
+        activation_dtype="e4m3_float8",
+        weight_dtype="e5m2_float8",
         storage_dtype="uint32",
         model_dtype="float16",
-        linear_weight_layout="NK",
+        no_scale=True,
```

Comment:
Is it needed or helpful to expose the `no_scale` in the config dictionary as opposed to just inferring based on the type? 

I also kind prefer to use if config.scale: as opposed to the negation but that's nit and I understand you were following convention here.